tools/pi_db_tools/pi_database_splitter.py

```diff
@@ -0,0 +1,179 @@
+# => python3.3 required!

```

Comment:
I would not call it with python3 ... this is up the the environment to provide you the correct python version ... there might be no python3 bin, but still python is using python3.